v1/metrics.yaml

```diff
@@ -339,6 +346,9 @@ paths:
                 x-client-ip: '{{x-client-ip}}'
       x-monitor: false
 
+  ########################################
+  # Legacy
+  ########################################
   /legacy/:
     get:
       tags:
```

Comment:
Not really related but I've just noticed that the listing endpoint and an actual endpoint have different tags so they show up in different sections in the documentation. Might be nice to fix as we're on it anyway

Reply:
Done !